scripts/scripts.js

```diff
@@ -38,12 +35,90 @@ const CONFIG = {
     pdfViewerReportSuite: 'adbadobenonacdcprod,adbadobedxprod,adbadobeprototype',
   },
   locales: {
+    // Americas
+    ar: { ietf: 'es-AR', tk: 'oln4yqj.css' },
+    br: { ietf: 'pt-BR', tk: 'inq1xob.css' },
+    ca: { ietf: 'en-CA', tk: 'pps7abe.css' },
+    ca_fr: { ietf: 'fr-CA', tk: 'vrk5vyv.css' },
+    cl: { ietf: 'es-CL', tk: 'oln4yqj.css' },
+    co: { ietf: 'es-CO', tk: 'oln4yqj.css' },
+    la: { ietf: 'es-LA', tk: 'oln4yqj.css' },
+    mx: { ietf: 'es-MX', tk: 'oln4yqj.css' },
+    pe: { ietf: 'es-PE', tk: 'oln4yqj.css' },
     '': { ietf: 'en-US', tk: 'hah7vzn.css' },
-    de: { ietf: 'de-DE', tk: 'hah7vzn.css' },
+    // EMEA
+    africa: { ietf: 'en', tk: 'pps7abe.css' },
+    be_fr: { ietf: 'fr-BE', tk: 'vrk5vyv.css' },
+    be_en: { ietf: 'en-BE', tk: 'pps7abe.css' },
+    be_nl: { ietf: 'nl-BE', tk: 'cya6bri.css' },
+    cy_en: { ietf: 'en-CY', tk: 'pps7abe.css' },
+    dk: { ietf: 'da-DK', tk: 'aaz7dvd.css' },
+    de: { ietf: 'de-DE', tk: 'vin7zsi.css' },
```

Comment:
Looks like you changed the de font file `hah7vzn.css` vs `vin7zsi.css`. Was this intentional?